platforms/gcp-dataproc-serverless-airgapped/spark_healthcare_nlp_serverless.py

```
@@ -0,0 +1,77 @@
+import json, os
+
+import sparknlp
+import sparknlp_jsl
+
+from sparknlp.base import *
```

Ruff (0.11.9)

`from sparknlp.base import *` used; unable to detect undefined names

(F403)